layouts/partials/post-info.html

```diff
@@ -15,15 +15,17 @@
     <div class="post-taxonomies">
         {{ if .Params.categories }}
             <ul class="post-categories">
-                {{ range .Params.categories }}
-                    <li><a href="{{ "categories/" | absLangURL }}{{ . | urlize }}">{{ . }}</a></li>
+                {{ range $cat_name := .Params.categories }}
+                    {{ $cat := ($.Site.GetPage (printf "/categories/%s" $cat_name)) }}
+                    <li><a href="{{ $cat.Permalink }}">#{{ $cat.Title | default $cat_name }}</a></li>
```

Comment:
Categories should be printed without # prefix

Reply:
Fixed, hashtag removed for category.